src/common/values.ts

```diff
@@ -126,41 +131,43 @@ export const writeValues = async (values: Record<string, any>, overwrite = false
   // and write to their files
   const promises: Promise<void>[] = []
   if (settings) promises.push(writeValuesToFile(`${env.ENV_DIR}/env/settings.yaml`, settings, overwrite))
-  if (secretSettings)
+  if (secretSettings || overwrite)
```

Comment:
Is it possible to move these checks inside writeValuesToFile?

Reply:
no, as the input is dependent on it